src/components/Layout.tsx

```diff
@@ -129,6 +135,22 @@ const Layout = ({ children }) => {
       <FooterContainer>
         <FooterComponent />
       </FooterContainer>
+      {introModalEnabled && (
+        <IntroModal>
+          <RichText
+            key={123}
+            html={
+              '<iframe ' +
+              'width="100%" ' +
+              'style="aspect-ratio: 16 / 9" ' +
+              'src="https://www.youtube.com/embed/MIdk7wCIdzY?autoplay=1" ' +
+              'title="Intro Video" ' +
+              'allowFullScreen>' +
+              '</iframe>'
+            }
```

Comment:
You can also use template literals (which allow line breaks) rather than concatenating strings, they're usually a bit easier to refactor and easier to read/ e.g:

```suggestion
            html={`<iframe width="100%" style="aspect-ratio: 16 / 9"
                src="https://www.youtube.com/embed/MIdk7wCIdzY?autoplay=1"
                title="Intro Video" allowFullScreen>
              </iframe>`}
```